packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnHeaderMenuItems.tsx

```diff
@@ -15,15 +15,15 @@ export function GridColumnHeaderMenuItems(props: GridColumnHeaderMenuItemsProps)
   const { hideMenu, currentColumn } = props;
   const handleListKeyDown = React.useCallback(
     (event: React.KeyboardEvent) => {
-      if (event.key === 'Tab') {
+      if (event.key === 'Tab' || event.key === 'Escape') {
         event.preventDefault();
         hideMenu();
       }
     },
     [hideMenu],
   );
   return (
-    <MenuList id="menu-list-grow" onKeyDown={handleListKeyDown}>
+    <MenuList aria-labelledby={`menu-button-${currentColumn.field}`} autoFocus autoFocusItem id="menu-list-grow" onKeyDown={handleListKeyDown}>
```

Comment:
not sure about the `menu-button` hardcoded prefix

Reply:
@dtassone thanks! any good suggestions?

Reply:
Why is the `id` here necessary, maybe I'm not seeing something?

Reply:
![image](https://user-images.githubusercontent.com/17064586/105692646-a02cd780-5f39-11eb-8862-567f8530d3ff.png)
![image](https://user-images.githubusercontent.com/17064586/105693035-2fd28600-5f3a-11eb-8e7f-f940dd9456f0.png)
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html
This document clarifies that `aria-labelledby` must be an ID reference, so i added a `id` to the column header filter button, and then used this ID as the `aria-labelledby` attribute of the column header MenuList

Reply:
We have a `useId` helper in the main repo, it's meant as a bridge until https://github.com/reactjs/rfcs/pull/32 is resolved.

Reply:
I have some doubts about how I can share this `id` between these two components if I use `useId()`? should i store `id` in `ColumnMenuState`?

Reply:
In theory, I would expect each button and menu to be rendered in the same component, so sharing the id should be possible because in the same scope.